app/js/components/topics/topicRuleList/topicRuleList.jade

```diff
@@ -52,12 +52,13 @@
         tbody
           tr.legend-item(ng-repeat="rule in filteredRules | filter:filterZero | orderBy: sorter.predicate : sorter.reverse track by rule.rule_id", ng-class="{acked: rule.acked}")
             td(ng-click='!rule.acked && viewImpactedSystems(topic.slug ? topic.slug : topic.id, rule.rule_id)')
-              a(ng-if='!rule.acked') {{rule.description}}
-              div(ng-if='rule.acked')
+              .rule-description(ng-if='rule.acked')
                 | {{rule.description}}
                 span.label.label-default.inline-left.rule-inactive(translate) INACTIVE
```

Comment:
we should have an incident-icon here as well in case the inactive rule is an incident.